### PR TITLE
Solar shading test cleanup

### PR DESCRIPTION
**Pull request by mjwitte**
Pull request overview
---------------------
 - Remove all differences between SolarShadingTest.idf and SolarShadingTestGPU.idf except the ShadowCalculation object.
 - Delete all SolarShadingTest* rvi files.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by Myoldmopar** [comment]
It will be interesting to see what CI shows, since the CSV files will be all sorts of different.

**Comment by mjwitte** [comment]
For SolarShadingTest it will choke and show no eso diffs, because the new variable is timestep and everything else is hourly or daily, So, the diff file will say
`Time series in <...\SolarShadingTest\eplusout.csv> and <...\SolarShadingTest\eplusout.csv> do not match` We may want to change that in both files to show everything at timestep?

File: testfiles/SolarShadingTestGPU.idf [hunk 1]
```diff
@@ -2211,7 +2211,7 @@
     Fraction,                !- Schedule Type Limits Name
     Through: 12/31,          !- Field 1
     For: AllDays,            !- Field 2
-    Until: 24:00,0.0;!0.2;        !- Field 3
```
Comment (mjwitte) [inline testfiles/SolarShadingTestGPU.idf:2214]: @mbadams5 Looks like this schedule has no effect with PixelCounting. Comparing the GPU file with the schedule as zero and 0.2 produces no change. The PolygonClipping file does show a difference with the schedule change.

**Comment by Myoldmopar** [comment]
Shading transmittance diffs due to the schedule change, some renumbering/reordering, and difference in audit due to report variable request differences.  Custom check failed in develop, and is now fixed there, so that's nothing to do with this.  I think this is good to go.  Should I dig deeper before merging @mbadams5 @mjwitte ?

**Comment by mjwitte** [comment]
@Myoldmopar You saw diffs from the schedule change? I did not locally on windows. Oh, wait, that's because CI reverts to PolygonClipping - no GPU. I'm fine with this as-is, unless you think we want to change the reporting frequencies to all be timestep.

**Pull request merged**